plugins/com.google.cloud.tools.eclipse.appengine.login.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google, Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.login
 Require-Bundle: org.hamcrest;bundle-version="1.1.0",
- org.junit;bundle-version="4.12.0"
+ org.junit;bundle-version="4.12.0",
+ com.google.cloud.tools.eclipse.test.dependencies
```

Comment:
Removed `com.google.cloud.tools.eclipse.test.dependencies` and added ` org.mockito.invocation;provider=google;version="1.10.19"` under `Import-Package`.